cmd/create/ocmrole/cmd.go

```diff
@@ -267,6 +267,10 @@ func run(cmd *cobra.Command, argv []string) {
 			})
 			os.Exit(1)
 		}
+		_, _, err = checkRoleExists(reporter, awsClient, roleNameRequested, isAdmin, aws.ModeManual)
```

Comment:
Can we move this before we generate the policy file so we don't generate  in case of run time error